### PR TITLE
Add handling for multiple xlsx files in zip

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: src/pulse3D/exceptions.py [hunk 1]
```diff
@@ -19,42 +19,34 @@ class PeakDetectionError(Exception):
...
+
+
+class IncorrectOpticalFileFormatError(Exception):
+    pass
+
+
 class TwoFeaturesInARowError(PeakDetectionError):
     """There must always be a peak in between valleys and vice-versa."""
 
-    def __init__(
```
Comment (NikitaPoly) [inline src/pulse3D/exceptions.py:25]: I think this is a formatting issue. Unless this is older code and its reformatting for the updated format.

Comment (luciipak) [reply src/pulse3D/exceptions.py:25]: My formatter removes the magic comma and formats to the line length of 110

File: src/pulse3D/plate_recording.py [hunk 2]
```diff
@@ -526,6 +527,25 @@ def _load_dataframe(self, df: pd.DataFrame) -> None:
                 stim_session = stim_session_raw[:, ~np.isnan(stim_session_raw[1])].astype(int)
                 wf.stim_sessions.append(stim_session)
 
+    def _load_optical_well_file(self, file_path: str, stiffness_factor: Union[int, None]):
+        self.is_optical_recording = True
+        if not self.wells:
```
Comment (tannermpeterson) [inline src/pulse3D/plate_recording.py:532]: When would `self.wells` be truthy here?

Comment (tannermpeterson) [inline src/pulse3D/plate_recording.py:532]: Couldn't we just set this to however many xlsx files are found in the glob?

File: src/pulse3D/plate_recording.py [hunk 3]
```diff
@@ -359,14 +358,16 @@ def __init__(
             with tempfile.TemporaryDirectory() as tmpdir:
                 zf = zipfile.ZipFile(path)
                 zf.extractall(path=tmpdir)
-                self.wells, calibration_recordings = load_files(
-                    tmpdir, stiffness_factor, inverted_post_magnet_wells
-                )
+
+                if glob.glob(os.path.join(tmpdir, "**", "*.h5"), recursive=True):
```
Comment (tannermpeterson) [inline src/pulse3D/plate_recording.py:362]: Should we enforce the zip files to only contains files?

Comment (luciipak) [reply src/pulse3D/plate_recording.py:362]: like instead of the `else` do `elif glob.glob(os.path.join(tmpdir, "*.xlsx"), recursive=True)`

**Review by tannermpeterson — APPROVED** [review]

**Pull request merged**